From 911038990df1357f452b0e38309261faf1de898f Mon Sep 17 00:00:00 2001
From: Mark Reynolds <mreynolds@redhat.com>
Date: Wed, 8 Aug 2018 17:19:27 -0400
Subject: [PATCH] Ticket 49893 - disable nunc-stans by default

Description:  Until nunc-stans is stablized we need to disable it

https://pagure.io/389-ds-base/issue/49893

Reviewed by: ?

(cherry picked from commit 2f2d3b1d7e7d847de1bb9ddf2f63e71dbc90f710)
---
 ldap/servers/slapd/libglobs.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ldap/servers/slapd/libglobs.c b/ldap/servers/slapd/libglobs.c
index 3bd5c1826..f8741028d 100644
--- a/ldap/servers/slapd/libglobs.c
+++ b/ldap/servers/slapd/libglobs.c
@@ -1681,7 +1681,7 @@ FrontendConfig_init(void)
     cfg->maxbersize = SLAPD_DEFAULT_MAXBERSIZE;
     cfg->logging_backend = slapi_ch_strdup(SLAPD_INIT_LOGGING_BACKEND_INTERNAL);
     cfg->rootdn = slapi_ch_strdup(SLAPD_DEFAULT_DIRECTORY_MANAGER);
-    init_enable_nunc_stans = cfg->enable_nunc_stans = LDAP_ON;
+    init_enable_nunc_stans = cfg->enable_nunc_stans = LDAP_OFF;
 #if defined(LINUX)
     init_malloc_mxfast = cfg->malloc_mxfast = DEFAULT_MALLOC_UNSET;
     init_malloc_trim_threshold = cfg->malloc_trim_threshold = DEFAULT_MALLOC_UNSET;
-- 
2.17.1

